sdk/ml/azure-ai-ml/tests/test_configs/compute/compute-ci-unit.yaml

```diff
@@ -1,6 +1,7 @@
 name: banchci
 type: computeinstance
 description: some_desc_ci
+location: eastus
```

Comment:
should we update docstrings in compute with the new behavior? Right now, the docstring for `Compute.location` reads:

```
:param location: The resource location, defaults to None
```

but it should probably be:

```
`defaults to workspace location`
```

Reply:
Good catch.